web/src/frontend/src/nav-soknad/redux/actions.ts

```diff
@@ -14,13 +17,25 @@ export type ActionTypes =
 export function setFaktumVerdi(
 	faktumKey: string,
 	value: FaktumValueType,
-	properties?: any
+	fakta: any
 ) {
-	return {
-		type: ActionTypeKeys.SET_FAKTUM_VERDI,
-		faktumKey,
-		value,
-		properties
+	const index: number = fakta.findIndex((item: Faktum) => {
+		return item.key === faktumKey;
```

Comment:
denne burde trekkes ut i en utilsfunksjon og returnere faktum med en gang i stedet for å slå opp indeksen

Reply:
setFaktumVerdi() tar nå faktum og verdi som innparametere i stedet.